src/Controller/Component/AuthorizationComponent.php

```diff
@@ -306,7 +342,7 @@ public function authorizeAction(): void
      * @param string $configKey Configuration key with actions.
      * @return bool
      */
-    protected function checkAction(string $action, string $configKey): bool
+    protected function actionConfigured(string $action, string $configKey): bool
```

Comment:
This method name reads a bit weird but since it's not a public method I don't think we need to bikeshed on it :)

Reply:
It was a quick change so didn't conflict. Hopefully something better comes up.